src/main/java/com/icthh/xm/uaa/domain/SocialUserConnection.java

```diff
@@ -15,15 +10,24 @@
 import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
 
 /**
  * A Social user.
  */
 @ToString
 @NoArgsConstructor
+@AllArgsConstructor
 @Entity
-@Table(name = "jhi_social_user_connection")
+@Table(name = "social_connection")
```

Comment:
Are these incremental changes? Or we should clear DB before the patch?

Reply:
db patch exists